st2common/st2common/models/utils/action_param_utils.py

```diff
@@ -79,7 +79,7 @@ def is_immutable(param_meta):
     return (required_params, optional_params, immutable_params)
 
 
-def cast_params(action_ref, params):
+def cast_params(action_ref, params, cast_overrides=None):
```

Comment:
Agreed. Happened during a major push to get the release out so slipped.

Next time I touch this file will get it done. 

Btw, this PR is closed so no real point in conversations here.